Pod/Classes/Categories/UIImage+DecodedImage.m

```diff
@@ -69,22 +83,91 @@ + (UIImage *)pin_decodedImageWithCGImageRef:(CGImageRef)imageRef
     }
     
     CGSize imageSize = CGSizeMake(CGImageGetWidth(imageRef), CGImageGetHeight(imageRef));
-    UIGraphicsBeginImageContextWithOptions(imageSize, opaque, 1.0);
-    CGContextRef ctx = UIGraphicsGetCurrentContext();
+    
+    CGBitmapInfo info = opaque ? (kCGImageAlphaNoneSkipFirst | kCGBitmapByteOrder32Host) : (kCGImageAlphaPremultipliedFirst | kCGBitmapByteOrder32Host);
+    CGColorSpaceRef colorspace = CGColorSpaceCreateDeviceRGB();
+    
+    //Use UIGraphicsBeginImageContext parameters from docs: https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIKitFunctionReference/#//apple_ref/c/func/UIGraphicsBeginImageContextWithOptions
+    CGContextRef ctx = CGBitmapContextCreate(NULL, imageSize.width, imageSize.height,
+                                             8,
+                                             0,
+                                             colorspace,
+                                             info);
+    
+    CGColorSpaceRelease(colorspace);
+    
```

Comment:
Is there an advantage to creating a bitmap context instead of letting UIMethod create one?